indico_themes_canonical/static/css/ubuntu-summit/header.css

```diff
@@ -47,22 +35,62 @@ Wraps around the title box
 */
 .confTitleBox {
   background: none;
+  display: flex;
+  margin: auto;
+  padding: 80px 20px;
   text-align: left;
-}
-
-.confTitle h1 {
-  width: auto;
+  width: var(--full-width);
 }
 
 /*
 The style for the title text.
 */
-.conference-title-link {
-  font-size: 2.228rem;
+.confTitle {
```

Comment:
Are these overrides of existing classes? the usual naming convention would be `conf-title`?

Reply:
Exactly this, these are to override previous classes